_layouts/base.html

```diff
@@ -22,7 +22,8 @@
 	<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
 	<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
 	<![endif]-->
-    <title>{{ site.title }}{% if page.title %}: {{ page.title }}{% endif %}</title>
+    <title>{{ site.data.lesson.title }}{% if page.title %}: {{ page.title }}{% endif %}</title>
+{% include schema_org.html %}
```

Comment:
How does it affect the header of a lesson?

Reply:
It does not affect the header.